### ISSUE TITLE
Unable to setup the integration

### ISSUE THREAD

**Issue by avhm** [amosyuen/ha-mikucare#26]
After cloning the repo using HACS and adding the integration the service fails with the error:

```
This error originated from a custom integration.

Logger: aiohttp.server
Source: custom_components/mikucare/config_flow.py:29 
Integration: Miku Care (documentation, issues) 
First occurred: 18:57:43 (4 occurrences) 
Last logged: 19:02:29

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 85, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 100, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 80, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 235, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 146, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 148, in post
    return await super().post(request)
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 72, in wrapper
    result = await method(view, request, data, *args, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 71, in post
    result = await self._flow_mgr.async_init(
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 845, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 873, in _async_init
    result = await self._async_handle_step(flow, flow.init_step, data)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 367, in _async_handle_step
    result: FlowResult = await getattr(flow, method)(user_input)
  File "/config/custom_components/mikucare/config_flow.py", line 61, in async_step_user
    return await self.async_step_auth(user_input)
  File "/config/custom_components/mikucare/config_flow.py", line 102, in async_step_auth
    data_schema=vol.Schema(_get_auth_schema(user_input)),
  File "/config/custom_components/mikucare/config_flow.py", line 29, in _get_auth_schema
    vol.Required(CONF_USERNAME, default=data.get(CONF_USERNAME, "")): str,
AttributeError: 'NoneType' object has no attribute 'get'
```

this manifests as
![IMG_3968](https://github.com/amosyuen/ha-mikucare/assets/271955/0b0f5234-f766-4207-8449-22e3a3e5502e)

Using:
Home Assistant 2023.5.4
Frontend 20230503.3 - latest

**Comment by amosyuen** [comment]
Thanks for catching, I forgot to publish the fix in [v1.0.2](https://github.com/amosyuen/ha-mikucare/releases/tag/v1.0.2) as official release. It is now published.

**Comment by avhm** [comment]
Smashing, works great. Thank you!
